### PR TITLE
Dev to master

### PR DESCRIPTION
**Pull request by yashodhicy**
project file reviewing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Ibizugbe) [inline .gitignore:1]: Nice job on creating the `test.md` file. 👍but just as [required in this milestone](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), the path to this file should be placed in your `.gitignore` file. when this is done, the `test.md` file will be ignored by git which stops it from being deployed to github😉 currently, your `test.md` file is still being deployed to github😉

ignored by git

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @yashodhicy  👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Github flow has been used👍.
- [x] `index.html` and `style.css` files have been created👍.


## Required Changes ♻️

- [x] We have learnt in the first point on the [5 basic rules of to follow in microverse projects](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) that we Use descriptive names for your branches (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars). Kindly checkout to a new branch with a discriptive name and open a PR with all the changes below implemented and ensure to reference this PR in your new PR description😉 a good branch name can be `hello-microverse-setup` 😉

- [x] Kindly use the[ microverse readme template](https://github.com/microverseinc/readme-template) and add a descriptive readme file personalised for this project😉 Currently, your readme file is not properly filled and it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that our project is documented professionally with the [microverse readme template](https://github.com/microverseinc/readme-template) as a guideline😉 Kindly read every instruction in the [template](https://github.com/microverseinc/readme-template) and create a personalised readme file for this project😉 every section in the template should be properly filled except the FAQ section which is optional.
- [x] Nice work on using some github flow practice😉, but let's do better by adding a descriptibe pull request title and summary😉 I suggest you take a look at this lesson for guidance and edit your pull request heading and summary to make it better😉
- [x] Kindly setup your linters correctly for this project as it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) in this milestone that your linters are properly setup and the test is passing😉
image

- [x] Nice work creating the `test.md` file as required😉, but it is also a requirement that this file is ignored by `git` hence it's path should be added to your `.gitignore` file and when ignored by git, it won't be deployed to github😉 Kindly create a `.gitignore` file and add the path to your `node modules` and `test.md`😉
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (yashodhicy) [reply .gitignore:1]: thank you

Comment (yashodhicy) [reply .gitignore:1]: @Ibizugbe thank you for all the support. I went through all the problems. But due to renaming the branch I had to open a new pull request. 
this is the link to the new pull request: https://github.com/yashodhicy/Microverseday2/pull/2

**Pull request closed without merging**